WalletWasabi.Tests/RegTests.cs

```diff
@@ -759,10 +757,9 @@ public async Task SendTestsAsync()
 			var nodes = new NodesGroup(global.Config.Network, requirements: Constants.NodeRequirements);
 			nodes.ConnectedNodes.Add(await RegTestFixture.BackendRegTestNode.CreateNodeClientAsync());
 
-			// 2. Create mempool service.
-			var mempoolService = new MempoolService();
+			// 2. // Dispose mempool.
```

Comment:
Should this comment say `Create mempool` or `Dispose mempool`?

Reply:
Fixed.